driver/syscall_table.c

```diff
@@ -26,11 +26,16 @@ or GPL2.txt for full copies of the license.
 #include <asm/syscall.h>
 #endif
 #else /* __KERNEL__ */
+
+#ifdef WDIG
```

Comment:
This was supposed to be an `ifndef` ?

We could do `#if defined(__linux__)`

Reply:
no, this is supposed to be an ifdef. wdig includes a syscall.h which includes some definitions that are missing in that environment.